modules/tinymce/src/core/main/ts/api/commands/NewlineCommands.ts

```diff
@@ -1,19 +1,20 @@
+import * as InsertBlock from '../../newline/InsertBlock';
 import * as InsertBr from '../../newline/InsertBr';
 import * as InsertNewLine from '../../newline/InsertNewLine';
 import Editor from '../Editor';
 
 export const registerCommands = (editor: Editor): void => {
   editor.editorCommands.addCommands({
     insertParagraph: () => {
-      InsertNewLine.insert(editor);
+      InsertNewLine.insertBreak(InsertBlock, editor);
     },
 
     mceInsertNewLine: (_command, _ui, value) => {
       InsertNewLine.insert(editor, value);
     },
 
-    InsertLineBreak: (_command, _ui, value) => {
-      InsertBr.insert(editor, value);
+    InsertLineBreak: (_command, _ui, _value) => {
+      InsertNewLine.insertBreak(InsertBr, editor);
```

Comment:
Also just a though... do we need a `Fixed` changelog entry for this?

Reply:
Ah probably, yes